addon/MetadataStore.js

```diff
@@ -17,6 +19,8 @@ XPCOMUtils.defineLazyModuleGetter(this, "OS",
 
 const METASTORE_NAME = "metadata.sqlite";
 
+const SQL_MIGRATIONS = migrations;
```

Comment:
Just wanted to keep the convention as the uppercase. I think I can rename the `migration` directly.